Rxavio/Teamwork
# #168497210  employee can sign up api

#### What does this PR do?
 -  signup endpoint
#### Description of Task to be completed?
-  To make this endpoint working correctly
POST/api/v1/auth/signup
#### How should this be manually tested?
- Using postman you should enter the required information in the body and click send button 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168497210
#### Questions: (edited) 
N/A
